inferentia/scripts/setup.sh

```diff
@@ -34,18 +34,20 @@ Sets up python execution environment for AWS Neuron SDK for execution on Inferen
 -v|--python-version        Python version, default is 3.7
 -i|--inferentia-path       Inferentia path, default is: /home/ubuntu
 -p|--use-pytorch           Install pytorch-neuron if specified
--t|--use-tensorflow        Install tensorflow-neuron if specified
+-t|--use-tensorflow        Install tensorflow-neuron is specified
+--tensorflow-version       Version of Tensorflow used. Default is 1. Ignored if installing pytorch-neuron
 "
 
 # Get all options:
-OPTS=$(getopt -o hb:v:i:tp --long help,python-backend-path:,python-version:,inferentia-path:,use-tensorflow,use-pytorch -- "$@")
+OPTS=$(getopt -o hb:v:i:tp --long help,python-backend-path:,python-version:,inferentia-path:,use-tensorflow,use-pytorch,tensorflow-version: -- "$@")
 
 
 export INFRENTIA_PATH=${TRITON_PATH:="/home/ubuntu"}
 export PYTHON_BACKEND_PATH="/home/ubuntu/python_backend"
 export PYTHON_VERSION=3.7
 export USE_PYTORCH=0
 export USE_TENSORFLOW=0
+export TENSORFLOW_VERSION="1"
```

Comment:
Why is this a string?

Reply:
We're checking against strings in the if statements